server/server.go

```diff
@@ -209,6 +214,16 @@ func CreateServer(ctx context.Context, cfg *config.Config, apiBuilders ...Handle
 			pdAPIPrefix: apiHandler,
 			webPath:     http.StripPrefix(webPath, ui.Handler()),
 		}
+
+		if cfg.EnableDashboard {
```

Comment:
Do we need to make it can be dynamic changed? if not, we can remove it and enable it in default, we already have a compiler option.  I more want to control some service runtime behavior, maybe do this later.

Reply:
It looks like the latest version has removed the compiler option

Reply:
I guess totally disabling a service is not very easy 🤔

Reply:
You can take a look at the current way of keyvisual，it's control with:
```
config set runtime-services keyvisual
```

Reply:
Will this configuration being syned into all PD nodes? If so, this might not be a good entry since we want to control the acess only for a single-instance basis. (Maybe we can copy the access control part?)

Reply:
ok, let's use this parameter temporarily.